serving/docker/partition/properties_manager.py

```diff
@@ -90,6 +91,21 @@ def set_and_validate_model_dir(self):
                     'Please specify the option.model_dir or option.model_id or include model files in the model-dir.'
                 )
 
+    def generate_checkpoints_json(self):
+        if self.properties['engine'] == 'DeepSpeed':
+            config_file = os.path.join(self.properties['save_mp_checkpoint_path'], 'ds_inference_config.json')
+            configs = {}
+            if glob.glob(config_file):
```

Comment:
Why use glob? are you just trying to figure out if the file exists?

Reply:
Yeah, Just to know file exists or not.